src/main/java/unicash/logic/parser/FindCommandParser.java

```diff
@@ -31,35 +36,58 @@ public class FindCommandParser implements Parser<FindCommand> {
      */
     public FindCommand parse(String args) throws ParseException {
         requireNonNull(args);
+
+        /* All prefixes are parsed first */
+        ArgumentMultimap argMultimapWithAllPrefixes = ArgumentTokenizer.tokenize(args, PREFIX_NAME,
+                PREFIX_DATETIME, PREFIX_AMOUNT, PREFIX_TYPE, PREFIX_CATEGORY, PREFIX_LOCATION);
+
+        /* If any of the invalid prefixes are present, an invalid command format message
+         * is displayed to the user, as opposed to the invalid prefixes themselves being
+         * parsed together with the valid prefixes as a standard field input. */
+        if (areAnyPrefixesPresent(
+                argMultimapWithAllPrefixes, PREFIX_DATETIME, PREFIX_AMOUNT, PREFIX_TYPE)) {
+
+            throw new ParseException(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
+        }
+
         ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(
```

Comment:
Could use the old `ArgumentMultimap` instead since you're guaranteed as this point to have only the `n/`, `l/`, and/or `c/` prefixes

Reply:
Nice catch, I've removed the additional argmultimap!